### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by TorbenVallund**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -189,6 +189,18 @@ The EncryptedCardData is encrypted according to the OAEP padding scheme. Once de
   }
 }
 
+### Failed callback
+
+In case that the user rejects the payment and the payment times out, a failed callback is sent to the FailedPaymentCallbackUrl url supplied in the initiate payment call.
```
Comment (madfrdanskebank) [inline README.md]: Did you mean 'or the payment times out'?

File: README.md [hunk 2]
```diff
@@ -189,6 +189,18 @@ The EncryptedCardData is encrypted according to the OAEP padding scheme. Once de
...
+
+```
+{
```
Comment (madfrdanskebank) [inline README.md]: Can't remember how the reset is documented, but you might want to indicate this is a response body with a status code like sample below. This is also fine. 

```
Status: 400
Body: 
{
    'code': '100'
    ...
}
```

Comment (TorbenVallund) [reply README.md]: It is a request body to the above url

File: README.md (hunk 1 above)
Comment (TorbenVallund) [reply README.md]: In the flow diagram it is "and", but there might be other cases.

**Comment by TorbenVallund** [comment]
OK

**Pull request merged**